VideoApp/VideoApp/Stores/Chat/ChatFileMessage.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright (C) 2021 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import TwilioConversationsClient
+
+class ChatFileMessage: ChatMessage {
```

Comment:
This and `ChatTextMessage` are for:

1. Abstract `TCHMessage` to provide good boundary between SDK integration and higher layers. This has benefits including make it clearer how SDK is used, easier to update SDK, and can be easily mocked by higher layers.
2. Handle the many optional message properties.
3. Would be very useful if in the future the app added a new custom message type that required additional decoding. `TCHMessage` only has `text` and `media` types built in. So if there are more types of messages or if JSON attributes are set and need to be processed this would be a really nice place to decode more details.

I know I'm not being very lenient by requiring no `nil` values but seems like a simple way to go for this app. And there is a little redundant storage but not a lot of file messages being sent probably. I could be talked out of these things and still thinking about it a bit.